laskyawm-bar.css

```diff
@@ -1,12 +1,12 @@
 #bar {
-	background-color: rgba(0, 0, 0, .25);
+	background-color: rgba(255, 255, 255, 0.6);
+	color: #000;
```

Comment:
I know I said make windows the same as the bar, but minimized windows are a little hard to distinguish from the bar now. Maybe give minimized windows a more opaque color and the bar a different color?